public/config.js

```diff
@@ -0,0 +1,3 @@
+window.__config__ = {
+  hello: "hello",
```

Comment:
?

Reply:
I tested whether I had injected this variable correctly into the index.html file. I removed it and performed a squash.